src/components/table-of-contents/_macro.spec.js

```diff
@@ -167,4 +194,49 @@ describe('macro: table-of-contents', () => {
             });
         });
     });
+
+    describe('related links and print button', () => {
```

Comment:
```suggestion
    describe('related links and button', () => {
```
Maybe remove "print" here because its not hardcoded to a print button anymore